--- /dev/shm/kdevelop/kdevelop.m4.in	2011-06-16 14:07:17.000000000 -0500
+++ /dev/shm/kdevelop/kdevelop.m4.in.new	2011-11-07 17:46:08.830760322 -0600
@@ -417,8 +417,8 @@
          db->open( db, 0, 0, NULL, DB_BTREE, DB_CREATE, 0644 );
       #endif
     ],
-    ac_cv_bdb=yes,
-    ac_cv_bdb=no
+    ac_cv_bdb=no,
+    ac_cv_bdb=yes
     )
     CXXFLAGS="$save_CXXFLAGS"
     LDFLAGS="$save_LDFLAGS"
